### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by lhmartin**
Changed the requirements to fix this issue. Just used the max one.

Screenshot 2019-11-20 at 12 47 46

### CODE REVIEW

**Comment by greenson** [comment]
Oh it is running pip of python2 rather than pip3!

I am investigating this issue as well. Maybe we are missing this file I guess?
https://github.com/nyu-devops/lab-bluemix-cf/blob/master/runtime.txt

**Comment by lhmartin** [comment]
Maybe, I can add that to this request.

**Comment by lhmartin** [comment]
> Maybe we are missing this file I guess?
> https://github.com/nyu-devops/lab-bluemix-cf/blob/master/runtime.txt

Added it now

File: requirements.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ gunicorn==19.9.0
 nose==1.3.7
 pinocchio==0.4.2
 factory-boy==2.12.0
-pylint>=2.4.1
+pylint>=1.9.5
```
Comment (greenson) [inline requirements.txt]: Let's change it back to the pip3 version :)

Comment (lhmartin) [reply requirements.txt]: Done so

**Pull request merged**